draft-ietf-quic-transport.md

```diff
@@ -466,16 +481,16 @@ Protected Payload:
 
 : Packets with a short header always include a 1-RTT protected payload.
 
-The header form and connection ID field of a short header packet are
-version-independent.  The remaining fields are specific to the selected QUIC
-version.  See {{QUIC-INVARIANTS}} for details on how packets from different
-versions of QUIC are interpreted.
+The header form and connection ID field, if present, of a short header packet
```

Comment:
A 0-byte CID is always present and the extra clause makes this read more awkwardly, so I'd prefer keeping this as is.